build.sbt

```diff
@@ -2,7 +2,9 @@ name := "PlayMapRouletteV2"
 
 version := "1.0"
 
-lazy val `playmaproulettev2` = (project in file(".")).enablePlugins(PlayScala)
+lazy val `playmaproulettev2` = (project in file("."))
+  .enablePlugins(PlayScala)
+  .enablePlugins(SbtWeb)
```

Comment:
On my local build I haven't had any issues with it, so probably something I should look into a little more. What problem exactly did the plugin resolve?